sw/host/opentitantool/src/command/spx.rs

```diff
@@ -58,7 +37,7 @@ impl CommandDispatch for SpxKeyShowCommand {
         _context: &dyn Any,
         _transport: &TransportWrapper,
     ) -> Result<Option<Box<dyn Annotate>>> {
-        let key = load_pub_or_priv_key(&self.key_file)?;
```

Comment:
As we discussed offline, the `spx` command should provide generic crypto operations independent of our manifest format. Since #18195 has those changes, I think the remaining changes in this file will probably conflict with those. Do you mind if we merge #18195 first?